roles/kubernetes/preinstall/tasks/verify-settings.yml

```diff
@@ -68,7 +68,7 @@
 # NOTICE: the check blatantly ignores the inet6-case
 - name: Guarantee that enough network address space is available for all pods
   assert:
-    that: "{{ kubelet_max_pods <= ((32 - kube_network_node_prefix) ** 2) - 2 }}"
+    that: "{{ kubelet_max_pods <= (2 ** (32 - kube_network_node_prefix)) - 2 }}"
```

Comment:
why not use:
`that: "{{ kubelet_max_pods <= kube_pods_subnet | ipaddr('net') | ipaddr('size') - 2 }}"`

Reply:
kube_pods_subnet - the CIDR range of pods in the cluster. Default 10.233.64.0/18 (16,382 ip's).

Reply:
oh yes, missed this